diff --git a/freescout/app/Misc/WpApi.php b/freescout/app/Misc/WpApi.php
index be289c2..5407436 100644
--- a/freescout/app/Misc/WpApi.php
+++ b/freescout/app/Misc/WpApi.php
@@ -102,7 +102,7 @@ class WpApi
      * Check module license.
      */
     public static function checkLicense($params)
-    {
+    {return;
         $params['action'] = self::ACTION_CHECK_LICENSE;
 
         $endpoint = self::ENDPOINT_MODULES;
@@ -118,7 +118,7 @@ class WpApi
      * Check module license.
      */
     public static function checkLicenses($params)
-    {
+    {return;
         $params['action'] = self::ACTION_CHECK_LICENSES;
 
         $endpoint = self::ENDPOINT_MODULES;
diff --git a/freescout/app/Module.php b/freescout/app/Module.php
index 63752f0..4e70b66 100644
--- a/freescout/app/Module.php
+++ b/freescout/app/Module.php
@@ -153,7 +153,7 @@ class Module extends Model
      * Deactivate module and update modules cache.
      */
     public static function deactiveModule($alias, $clear_app_cache = true)
-    {
+    {return;
         self::setActive($alias, false);
         // Update modules cache
         \Module::clearCache();
